static/js/common/upload-addon.js

```diff
@@ -78,6 +78,11 @@
             $upload_field.wrap(ui_parent);
             $upload_field.before(ui_link);
             $upload_field.parent().after(ui_details);
+            $upload_field.after($('#wizardlink'));
+
+            $('#wizardlink').on('click', function() {
+                window.location = this.href;
```

Comment:
Is this necessary because of other js interaction with fake buttons?

Reply:
Yeah, the fake file upload field we use gets in the way.  Ideally it'd just be in a different div but if we want it equal and next to the upload button it has to be in the same div, alas.